### PR TITLE
[FIZZ-100] Standardize the Sections output file

### PR DESCRIPTION
**Pull request by stephenfuqua**

### CODE REVIEW

File: src/schoology-extractor/schoology_extractor/mapping/sections.py [hunk 1]
```diff
@@ -0,0 +1,65 @@
...
+    df = sections_df[["id", "section_title", "description", "active"]].copy()
+
+    df["SourceSystem"] = constants.SOURCE_SYSTEM
+    df["EntityStatus"] = constants.ACTIVE
+
+    df.rename(
+        columns={
+            "id": "SourceSystemIdentifier",
+            "section_title": "Title",
+            "description": "SectionDescription",
```
Comment (stephenfuqua) [inline src/schoology-extractor/schoology_extractor/mapping/sections.py:50]: On lines 43 and 44 I used constants to represents _values_, but elsewhere I have used plain strings to represent _column names_. This just felt natural as I was writing it. Are you guys comfortable with that, or do you find it to be weirdly inconsistent?

@bradbanister  @lalrg

File: src/schoology-extractor/schoology_extractor/mapping/sections.py [hunk 2]
```diff
@@ -0,0 +1,65 @@
...
+
+    Parameters
+    ----------
+    sections_df: DataFrame
+        Pandas DataFrame containing Schoology sections
+
+    Returns
+    -------
+    DataFrame
+        A LMSUsers-formatted DataFrame
```
Comment (stephenfuqua) [inline src/schoology-extractor/schoology_extractor/mapping/sections.py:24]: Oops, need to change that from "Users" to "Sections"

File: src/schoology-extractor/schoology_extractor/mapping/sections.py (hunk 1 above)
Comment (bradbanister) [reply src/schoology-extractor/schoology_extractor/mapping/sections.py:50]: 👍  I find it natural as well

File: src/schoology-extractor/schoology_extractor/main.py [hunk 3]
```diff
@@ -85,17 +85,21 @@ def _get_users():
...
+    result_bucket["sections"] = sections
     return sections
 
 
 def _get_assignments(section_id) -> Callable:
     def __get_assignments():
         assignments = service.get_assignments(section_id)
         result_bucket["assignments"] = assignments
-        return service.map_assignments_to_udm(assignments, section_id) if not assignments.empty else None
+        return (
```
Comment (bradbanister) [inline src/schoology-extractor/schoology_extractor/main.py:98]: what are the types being returned?

**Review by bradbanister — APPROVED** [review]
Looks good.  Suggest adding return type on one function if possible.

File: src/schoology-extractor/schoology_extractor/mapping/sections.py (hunk 1 above)
Comment (lalrg) [reply src/schoology-extractor/schoology_extractor/mapping/sections.py:50]: Looks good to me

File: src/schoology-extractor/schoology_extractor/main.py (hunk 3 above)
Comment (stephenfuqua) [reply src/schoology-extractor/schoology_extractor/main.py:99]: Done

File: src/schoology-extractor/schoology_extractor/mapping/sections.py (hunk 1 above)
Comment (stephenfuqua) [reply src/schoology-extractor/schoology_extractor/mapping/sections.py:50]: OK, thanks for confirming.

**Review by lalrg — APPROVED** [review]
I run the tests, reviewed the code, and tested the functionality and it works as expected.

**Pull request merged**